Show error on kubernetes pages and kubernetes endpoint page.

KlapTrap opened the pull request:
Made new smaller effect that gets all error wrapper events and throws an internal error based on some new config.

I've also changed the routing path variable to be common across kubernetes and cf pages to make it easier for the page header to automatically pickup endpoint error states.

Commit:
Wire in internal errors into api error wrapper

Commit:
Use common endpoint id routing variable for easier header errors

Commit:
Fix error where header error message doesn't show correctly

richard-cox committed:
Ensure internal error status is string

Commit:
Clean up kubernetes entities when disconnecting k8s endpoint type

richard-cox committed:
Add internal error for non-passthrough cf calls

richard-cox committed:
Ensure all status codes are strings

richard-cox commented on src/frontend/app/store/reducers/pagination-reducer/pagination-reducer-clear-pagination-type.ts:
```diff
@@ -9,29 +19,55 @@ import {
+      const clearedEntity = Object.keys(entityState).reduce((prevEntityState, key) => {
+        return {
+          ...prevEntityState,
+          [key]: defaultPaginationEntityState
```
Regarding the previous code.. when we come back to this in the future we should just remove the entry instead of resetting.

richard-cox reviewed (approved):
Looks good, once we get it passed gates this can be merged

richard-cox committed:
Merge remote-tracking branch 'origin/v2-master' into kube-endpoint-errors

richard-cox committed:
Fix more `cfId` --> `endpointId`

richard-cox committed:
Improve e2e tests
- extend time for create service instance tests
- extend time for some user tests (don't think this will help as much as SI)
- ensure we don't fail when a service we don't want to be there doesn't
